draft-ietf-rtcweb-jsep.xml

```diff
@@ -2293,6 +2284,14 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             section is bundled into another m= section, it still MUST
             NOT contain any ICE credentials.</t>
 
+            <t>If the m= section is not bundled into another
```

Comment:
This whole section discusses the pre-answer state, so I think you want to omit the "answer rejected RTCP mutiplexing" clause. You can then add text below to say this isn't needed once the answer chooses muxing.